clientSide/womeneducation/src/Components/Home/HomeBackEnd.css

```diff
@@ -0,0 +1,212 @@
+.nav
+{
+    width:80px;
+    border:3px solid rgb(20, 20, 20);
+    height:1200px;
+    position: fixed;
+    left: 0px;
+    top:0px;
+    background-color: aliceblue;
+}
+.navbar
+{
+    width:70px;
+    height:70px;
+    margin-top: 20px;
+    margin-left: 3px;
+}
+.navbar1
+{
+    width:70px;
+    height:70px;
+    margin-top: 300px;
+    margin-left: 3px;
+}
+.box
+{
+    width:1360px;
+    height:1200px;
+    margin-left: 83px;
+    /* border: 5px solid red; */
+}
+.jtdimagesize
+{
+    width:60px;
+    height:60px; 
+}
+.imagesize1
+{
+    width:70px;
+    height:60px;
+    margin-top: 3px;
+}
+.imagesize2
+{
+    width:70px;
+    height:70px;
+    margin-top: 150px;
```

Comment:
.imagesize1 and imagesize2 have similar properties 

please use one class name

Reply:
I removed same classname